README.md

```diff
@@ -102,7 +107,13 @@ Each file should look as follows:
 | 9.8 | 0.045
 | ... | ...
 
-### Running the script
+#### template_list.csv
+The `template_list.csv` file contains the following columns:
+
+ * **session_id**. Here you can specify the API session ID of the scenario template. 
```

Comment:
I think the terms "API session ID" and "session_id" are not immediately clear to users not familiar with the API. Maybe simply use "id" just like the scenario_list.csv?